docs/firefighting/index.md

```diff
@@ -0,0 +1,108 @@
+This page summarizes how to do various things related to HQ.
+For a more user-friendly guide check out [Cory's brown bag on the topic](http://prezi.com/wedwm-dgqto7/firefighting-hq/).
+
+# HQ Architecture and Machines
+
+![](./hq_architecture.jpg)
+
+# High-level System Monitoring and Alerts
+
+[HQ Vitals](https://app.datadoghq.com/dashboard/g9s-pw6-tpg/hq-vitals?to_ts=1549314000000&is_auto=false&from_ts=1549227600000&live=true&tile_size=m&page=0) - Various graphs on datadog
+
+[Datadog Alerts](https://app.datadoghq.com/monitors/manage?q=status%3A(alert%20OR%20warn%20OR%20"no%20data")) - these are also often reported on #hq-ops or #hq-ops-priority on slack
+
+https://www.commcarehq.org/hq/admin/system/ - catchall system info, contains deploy history, pillowtop info, and a bunch of other stuff
```

Comment:
HQ links are listed as variables below:
```https://[commcarehq.environment.path]/hq/admin/system/check_services```
The link to prod is ever-so-slightly more useful, but I don't really care.  Flagging in case you feel like making that consistent.

Reply:
Didn't want to make any changes to this or establish conventions in this PR